pkg/Cpanel/Security/Advisor/Assessors/Symlinks.pm

```diff
@@ -57,9 +58,9 @@ sub generate_advice {
 
 sub has_cpanel_hardened_kernel {
     my $self                  = shift;
-    my $hardened_kernel_state = $self->_check_for_symlink_kernel_patch();
+    my $hardened_kernel_state = ( Cpanel::Sys::Uname::get_uname_cached() )[2];
```

Comment:
This change removes the only usage of `_check_for_symlink_kernel_patch()`; the subroutine should be removed to avoid technical debt.

Reply:
And if we remove `_check_for_symlink_kernel_patch()`, we should also remove the `_enforcing_symlink_ownership()` and `_symlink_enforcement_gid()` functions, plus the `use Lchown ()`, `use Cpanel::TempFile ()`, `use Cpanel::GenSysInfo ()`, and `use Cpanel::Config::Httpd ()` lines.

Reply:
ok